cmd/consoleplugin/main.go

```diff
@@ -94,30 +85,7 @@ func main() {
 		os.Exit(1)
 	}
 
-	cl, err := client.New(cfg, client.Options{
-		Scheme: runtimeScheme,
-	})
-	if err != nil {
-		setupLog.Error(err, "creating a new client failed")
-		os.Exit(1)
-	}
-
-	config, err := membercfg.GetConfiguration(cl)
-	if err != nil {
-		setupLog.Error(err, "Error retrieving Configuration")
-		os.Exit(1)
-	}
-
-	pluginServer := startConsolePluginService(config.WebConsolePlugin())
-
-	gracefulShutdown(gracefulTimeout, pluginServer)
-}
-
-func startConsolePluginService(config membercfg.WebConsolePluginConfig) *consoleplugin.Server {
-	consolePluginServer := consoleplugin.NewConsolePluginServer(config, setupLog)
-	consolePluginServer.Start()
-
-	return consolePluginServer
+	gracefulShutdown(gracefulTimeout)
```

Comment:
Why do you keep it? We don't really care about graceful shutdown of the web console service, do we? Why not deleting the entire `cmd/consoleplugin/main.go` and then manually deleting already running web console plugin service/apps in clusters?

Reply:
right, I missed that 🤦‍♂️

Reply:
removed in e99f22c